tools/lib/perlcritic/Perl/Critic/Policy/HashKeyQuotes.pm

```diff
@@ -22,10 +22,11 @@ sub violates {
     return if !is_hash_key($elem);
 
     my $c = $elem->content;
-    # special characters
-    return if $c =~ m/[- \/<>.=_:\\\$]/;
+    # Quotes allowed, if not matching following regex
+    # TODO: PERL allow _ in simple identifiers, so we should do it too
```

Comment:
Please don't add TODOs in commits. Either you see it as important enough for a ticket or just leave it out

Reply:
ok
https://progress.opensuse.org/issues/58100

Reply:
Please don't use imperative form here.
And on top I disagree with you. TODOs are just fine from my pov.

Reply:
Of course feel free to disagree. I am glad to receive your opinion regardless which direction it takes :) I am referring to what is commonly done in this project and what I and others sticked to in before.